parser/src/layout.rs

```diff
@@ -173,6 +173,7 @@ where
         loop {
             // Retrieve the current indentation level if one exists
             let offside = match (&token.value, self.indent_levels.last().cloned()) {
+                (&Token::ShebangLine(_), _) => return Ok(token),
```

Comment:
Yeah, the layout caused me a bit of grief since it auto inserted a Semi token which caused an Unexpected Token error.  I thought I was misunderstanding how lalrpop was supposed to work for a little bit.

I'm kind of wondering if it wouldn't make sense to have the tokenizer tests use the Layout so this type of issue is more easily visible.